packages/dds/tree/src/simple-tree/api/typesUnsafe.ts

```diff
@@ -150,6 +150,17 @@ export type TreeFieldFromImplicitFieldUnsafe<TSchema extends Unenforced<Implicit
  */
 export type AllowedTypesUnsafe = readonly LazyItem<TreeNodeSchemaUnsafe>[];
 
+/**
+ * {@link Unenforced} version of {@link ImplicitAllowedTypes}.
+ * @remarks
+ * Do not use this type directly: it is only needed in the implementation of generic logic which define recursive schema, not when using recursive schema.
```

Comment:
```suggestion
 * Do not use this type directly: it is only needed in the implementation of generic logic which define recursive schema, not when using recursive schema.
 * @privateRemarks
 * This is similar to `Unenforced<ImplicitAllowedTypes>` in that it avoids constraining the schema
  * (which is necessary to avoid breaking recursive types),
  * but is superior from a safety perspective because it constrains the structure containing the schema.
```